nightingale_utils/configs/robot_configuration_lookup.yaml

```diff
@@ -0,0 +1,18 @@
+robot_configurations: {
```

Comment:
Can we change this to `joint_configurations`. I expect to have a `cartesian_configurations` that does something similar to what you have here, except in cartesian space.

Reply:
how about "home_joint_configurations"? these values represent "home".